AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/ActivityWithServices.java

```diff
@@ -279,4 +279,28 @@
 	@SystemService
 	WindowManager windowManager;
 
+	AppWidgetManager methodInjectedAppWidgetManager;
```

Comment:
i just realized that there is no consistency at all in the test classes. the `BeanInjctedActivity` has all fields `public`